### PR TITLE
[IMP] theme_*: adapt themes with new `s_motto` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
*: (aviato, bistro, kea, kiddo, nano, orchid, paptic, vehicle)

This commit adapts the design of new `s_motto` snippet for multiple themes.

task-4077607

requires: https://github.com/odoo/odoo/pull/175550

### CODE REVIEW

**Review by stefanorigano — APPROVED** [review]
LGTM, thanks 👍

File: theme_vehicle/views/customizations.xml [hunk 1]
```diff
@@ -348,25 +348,11 @@
...
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[4]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[5]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[6]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
```
Comment (SergeBayet) [inline theme_vehicle/views/customizations.xml:369]: @anso-odoo I don't see this in the task but I suppose that was discussed.

**Review by SergeBayet — APPROVED** [review]
LGTM! :+1:

File: theme_vehicle/views/customizations.xml (hunk 1 above)
Comment (anso-odoo) [reply theme_vehicle/views/customizations.xml:369]: Ooops, no, I think it's a conflict fix issue. Will fix this 👍

Comment (SergeBayet) [reply theme_vehicle/views/customizations.xml:369]: @anso-odoo Ok then :)

Comment (anso-odoo) [reply theme_vehicle/views/customizations.xml]: fixed ✅

**Comment by sobo-odoo** [comment]
@robodoo delegate=SergeBayet

**Review by SergeBayet — APPROVED** [review]

**Pull request closed without merging**